libs/features/georoutingv2/georoutingv2.css

```diff
@@ -1,15 +1,21 @@
 .dialog-modal.locale-modal-v2 .georouting-wrapper {
+  display: block !important;
   padding: 56px 24px 40px;
 }
 
 .dialog-modal.locale-modal-v2 {
-  background-image: url('/libs/features/georoutingv2/img/GeoModal_BG_Map_Mobile.png');
```

Comment:
the background image isn't positioning quite right with this solution. In addition, you could also get this to still work by just referencing an svg here instead of a png. That gets you the same awesome result. 🙌 
I would recommend reverting all this css code and just referencing an svg.

Reply:
What device or breakpoint are you are you looking at?

Reply:
The iPhone 12 Pro for example. You'll see the background image should scale to take up all the space in the background
So here's how it looks on main currently: 
![image](https://github.com/adobecom/milo/assets/115231412/b7d75c33-e75f-4684-98fc-0ab86251c543)

And here's how it looks on the branch:
![image](https://github.com/adobecom/milo/assets/115231412/e55b3f30-036a-4497-8ce7-2422503b7418)

Reply:
Good catch! Updated